.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode
 node_modules
 dist
-.DS_Store
\ No newline at end of file
+.DS_Store
+_metadata
```

Comment:
What is purpose of `_metadata` directory?

Reply:
That folder was automatically generated after I added the rules.json file to be used only in Chrome/Chromium. The subfolder `generated_indexed_rulesets` that contains the file `_ruleset1.json` is mentioned here: https://chromium.googlesource.com/chromium/src/+/HEAD/extensions/common/constants.cc